verification/curator-service/ui/src/components/SourceTable.tsx

```diff
@@ -18,7 +18,6 @@ import ParsersAutocomplete from './ParsersAutocomplete';
 import SourceRetrievalButton from './SourceRetrievalButton';
 import TextField from '@material-ui/core/TextField';
 import axios from 'axios';
-import { isUndefined } from 'util';
```

Comment:
Is this util used elsewhere or can it be removed?

Reply:
I can't find any other occurences.
Not sure where this was coming from.

![image](https://user-images.githubusercontent.com/1255432/94139907-b7b06280-fe6a-11ea-8594-7952ac57b227.png)